packages/integration/src/helpers.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Checks whether the given url is a valid host */
+export function isValidHost(url: string): boolean {
+  const check = new URL('http://example.com');
+  check.host = url;
```

Comment:
does this never-ever throw? :) interesting use of core facilities

Reply:
Nah, just silently ignores any invalid input :>

Reply:
i realised that it's a bit harsh to reject punycode things ('fågel.se' becoming 'xn--fgel-qoa.se' which is absolutely fine), hmm

Reply:
It does support it, but you need to enter the punycode version in config. That's how the implementation works anyway so should have the same validation in the config

Reply:
that makes sense!